src/menu.cpp

```diff
@@ -77,6 +77,21 @@ void Menu::readSections() {
 	addSection("settings");
 	addSection("applications");
 
+	/* To keep things simple, we will always create the applications section */
+	if (gmenu2x->hideSectionApplications)
+	{
+		auto itr = std::find(sections.begin(), sections.end(), "applications");
+		if (itr != sections.end())
+			sections.erase(itr);
+	}
+
+    if (gmenu2x->hideSectionGames)
```

Comment:
I see now that section Apps\Settings is always created if nonexistent, so maybe we could add simply a setting to hide them if requested (still I see no reason to hide Games). On a second look we could check if it's prefixed with dot (.) and then hide - would be useful in the long run